voxpupuli-puppet-lint-plugins.gemspec

```diff
@@ -2,20 +2,19 @@
 
 Gem::Specification.new do |s|
   s.name        = 'voxpupuli-puppet-lint-plugins'
-  s.version     = '4.0.0'
+  s.version     = '5.0.0.rc.1'
```

Comment:
please don't bump this. We will do this during the release process.

Reply:
Got it

Reply:
fixed